lib/slim_fast/parser.ex

```diff
@@ -105,13 +90,47 @@ defmodule SlimFast.Parser do
     end
   end
 
-  defp parse_line(@blank, _line), do: @blank
-  defp parse_line(@content, line), do: line |> String.slice(1..-1) |> String.strip |> parse_eex_string
-  defp parse_line(@comment, line), do: line |> String.slice(1..-1) |> parse_comment
-  defp parse_line(@html, line), do: line |> String.strip |> parse_eex_string
+  defp parse_attributes(""), do: {"", []}
```

Comment:
@Rakoth are you aware if there is any benefit to updating the code to use the module attribute `@blank` for blank strings like such: 

```elixir
defp parse_attributes(@blank), do: {@blank, []}
```